lib/Pipelines/PipelineRegistration.cpp

```diff
@@ -34,6 +38,15 @@ using mlir::func::FuncOp;
 
 namespace mlir::heir {
 
+void tosaToLinalg(OpPassManager& manager) {
```

Comment:
i *think* we removed all the TOSA in e2e tests and replaced them w/ linalg - which ones broke that you needed code? I can fix in a follow-up